bip-0322.mediawiki

```diff
@@ -76,16 +88,33 @@ The <code>to_sign</code> transaction is:
 
 A full signature consists of the base64-encoding of the <code>to_sign</code> transaction in standard network serialisation once it has been signed.
 
+This signature, as-is, only proves a message is signed by the intended recipient of the given invoice address which creates the <code>message_challenge</code> script.
+It does not prove any control of bitcoins or having had sent any transactions.
+
+TODO: How does this interact with as-of-yet-unspecified "Silent Transactions"?
```

Comment:
This isn't "future work", but a matter of compatibility, which BIPs should address.